Bottom: 6047896d1386ec4f1e0428d187e5e16fe85169a4
Top:    a61dda0a9f3595c9be0f724e0be46e363347f304
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 16:14:27 +0000

Enable -Wextra (NOT yet -pedantic)


---

diff --git a/configure.ac b/configure.ac
index ab8e873..b90115a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1143,7 +1143,7 @@ LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_L
 # if we have gcc then add -Wall
 if test "x$GCC" = "xyes"; then
 	# see about adding some extra checks if the compiler takes them
-	for flag in -Wall -fno-omit-frame-pointer; do
+	for flag in -Wall -Wextra -fno-omit-frame-pointer; do #-pedantic
 		case " ${CFLAGS} " in
 			*\ ${flag}\ *)
 				# flag is already present
